satpy/writers/cf_writer.py

```diff
@@ -215,8 +215,15 @@ def laea2cf(area):
 def create_grid_mapping(area):
     """Create the grid mapping instance for `area`."""
     try:
-        grid_mapping = mappings[area.proj_dict['proj']](area)
-        grid_mapping['name'] = area.proj_dict['proj']
+        try:
+            # let pyproj do the heavily lifting
+            # pyproj 2.0+ required
+            grid_mapping = area.crs.to_cf()
+            # not sure there is a better "standard" way of doing this
+            grid_mapping['name'] = grid_mapping['grid_mapping_name']
+        except AttributeError:
+            grid_mapping = mappings[area.proj_dict['proj']](area)
+            grid_mapping['name'] = area.proj_dict['proj']
```

Comment:
Does that mean that the `name` attribute would depend on the pyproj version? If so, should we align those names?

Reply:
My suggestion for 3 is not implemented. Right now this does not depend on the pyproj version because `proj` is statically defined and unchanging (`merc`, `geos`, etc). However, the `name` attribute is not required by CF for the grid mapping variable so we can remove this portion and instead use `area.area_id` as the name of the variable.

Reply:
Ok, I see. Using `area.area_id` sounds good!